roles/openshift_node/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 
 - name: Install sdn-ovs package
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-sdn-ovs{{ openshift_pkg_version | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
-  when: openshift.common.use_openshift_sdn and not openshift.common.is_containerized | bool
+  when: openshift.common.use_openshift_sdn and not openshift.common.use_flannel | bool and not openshift.common.is_containerized | bool
```

Comment:
@abutcher its always hard to remember what all options we need to set to make it work. To make it more friendly can we have one check in `roles/openshift_common/tasks/main.yml` to display a message saying make `openshift_use_openshift_sdn=false if you want to use flannel` or something?